cypress/integration/0-v2-markets/0-main-v2-market/details.page.aave-v2.spec.ts

```diff
@@ -0,0 +1,55 @@
+import { configEnvWithTenderlyMainnetFork } from '../../../support/steps/configuration.steps';
+import { skipState } from '../../../support/steps/common';
+import { supply, borrow } from '../../../support/steps/main.steps';
+import constants from '../../../fixtures/constans.json';
+import assets from '../../../fixtures/assets.json';
+
+const testData = {
+  depositETH: {
+    asset: assets.aaveMarket.ETH,
+    amount: 0.5,
+    hasApproval: true,
+  },
+  testCases: {
+    deposit: {
+      asset: assets.aaveMarket.USDT,
+      amount: 50,
+      hasApproval: false,
+    },
+    borrow: [
+      {
+        asset: assets.aaveMarket.USDT,
+        amount: 50,
+        apyType: constants.borrowAPYType.variable,
+        hasApproval: true,
+      },
+    ],
+  },
+};
+describe('Verify supply and borrow modals does appear', () => {
+  const skipTestState = skipState(false);
+  configEnvWithTenderlyMainnetFork({});
+  supply(testData.depositETH, skipTestState, true);
+  testData.testCases.borrow.forEach((borrowCase) => {
+    borrow(borrowCase, skipTestState, true);
+  });
+
+  describe('Verifing collateral, e-mode and isolation strings on detail page USDT AAVE V2', () => {
```

Comment:
wrong describe `e-mode and isolation strings on detail page USDT AAVE V2`
1. u don't check that
2. v2 version not contains option for e-mode and isolated mode